_pytest/nose.py

```diff
@@ -66,7 +66,7 @@ def is_potential_nosetest(item):
 def call_optional(obj, name):
     method = getattr(obj, name, None)
     isfixture = hasattr(method, "_pytestfixturefunction")
-    if method is not None and not isfixture and py.builtin.callable(method):
+    if method is not None and not isfixture and six.callable(method):
```

Comment:
i believe the callable builtin is back on all supported python versions so should be able drop the six reference here